Gems/LyShine/Code/Source/World/UiCanvasOnMeshComponent.cpp

```diff
@@ -36,74 +30,6 @@
 ////////////////////////////////////////////////////////////////////////////////////////////////////
 namespace
 {
-#if UI_CANVAS_ON_MESH_DEBUG
```

Comment:
I don't know if its wise to remove this, it can still be useful to toggle this debug code

Reply:
this is code for CryPhysics, I don't think that exists anymore?

https://github.com/aws/lumberyard/blob/master/dev/Code/CryEngine/CryPhysics/physicalworld.h

Reply:
Ok yeah I checked lumberyard source and the stuff calling this debug code is not there anymore (It's a shame that this code was not ported over, someone needed that in production at some point, and someone will likely need it again in the future)